tekton/resources/images/overlays/multi-arch/image-build-trigger-template.yaml

```diff
@@ -0,0 +1,70 @@
+- op: add
+  path: "/spec/resourcetemplates/0/spec/taskSpec/steps"
+  value:
+  - image: gcr.io/tekton-releases/dogfooding/buildx-gcloud:mario
```

Comment:
The `mario` tag is just for testing, we should not use it in here. Since we do nightly builds, you could use `latest` or a specific nightly tag/sha

Reply:
we don't have nightly build so far, as https://github.com/tektoncd/plumbing/pull/627 is not merged :(
so `gcr.io/tekton-releases/dogfooding/buildx-gcloud:mario` is only one publicly available version